src/main/groovy/org/shipkit/internal/notes/header/HeaderRemover.java

```diff
@@ -0,0 +1,46 @@
+package org.shipkit.internal.notes.header;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HeaderRemover {
+
+    public void removeHeaderIfExist(File releseNoteFile) {
+        try {
+            File tempFile = File.createTempFile("tmp", "shipkit");
+
+            BufferedReaderWrapper reader = new BufferedReaderWrapper(releseNoteFile);
```

Comment:
Reading the code, I don't see the value of the wrapper types? Is it only to handle the exception? The entire block is within try-catch anyway :)

If the wrapper types are only for handling exceptions, it does not seem to be enough ROI for keeping the code IMHO.